qir/qsharp/NegativeBaseProfiles/InternalCall/QSharpVersion/Example.qs

```diff
@@ -0,0 +1,16 @@
+namespace LoopCase {
```

Comment:
Here too, `LoopCase` seems to be the wrong name.

Reply:
Name updated.